project.godot

```diff
@@ -245,6 +245,12 @@ toggle_fullscreen={
 , Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":16777254,"unicode":0,"echo":false,"script":null)
  ]
 }
+e_cancel_move={
```

Comment:
Now that this has been assigned right click, I think it makes sense to generalize this as the cancel current action key previously mentioned by @hhyyrylainen.

Reply:
I went ahead and changed it.